network/alsp/params.go

```diff
@@ -0,0 +1,30 @@
+package alsp
+
+// To give a summary with the default value:
+//  1. The penalty of each misbehavior is 0.01 * misbehaviorDisallowListingThreshold = -86.4
+//  2. The penalty of each misbehavior is decayed by a decay value at each decay interval. The default decay value is 1000.
+//     This means that by default if a node misbehaves 1000 times in a second, it gets disallow-listed, and takes 86.4 seconds to recover.
+//     We emphasize on the default penalty value can be amplified by the engine that reports the misbehavior.
+//  3. Each time a node is disallow-listed, its decay speed is decreased by 90%. This means that if a node is disallow-listed
```

Comment:
Do we ever remove history of a past block? Depending on the likelihood of false positives we observe, it might be worthwhile to add this eventually. Otherwise a few false positives over a long period of time could cause significant disruption.

Reply:
In the mature implementation, the maximum disallow-listing interval is set to 24 hours (one day). However, until the implementation has been thoroughly tested and proven reliable, we will not enable a decrease in decay speed. Instead, we will allow every disallowed-listed node to be re-allow-listed after just a few seconds. This approach enables us to detect and disrupt potential attacks (with the manual intervention of the involved node operator) without being affected by false-positive cases.

As our implementation gains more real-world testing experience and edge cases are addressed, we plan to gradually decrease the decay speed for active spammers, taking into account the proportion of permissionless nodes. This will help us maintain a secure and efficient system while minimizing potential disruptions.